### ISSUE TITLE
Position xAxis labels on iOS

### ISSUE THREAD

**Issue by she-dev** [wuxudong/react-native-charts-wrapper#31]
Hello!

I'd like some help to get the labels onscreen.  A little more to the left and it would be perfect.
I checked [Charts wiki](http://cocoadocs.org/docsets/Charts/3.0.2/Classes/AxisBase.html#/s:vC6Charts8AxisBase8spaceMinSd) and tried setting spaceMin and spaceMax but it didn't seem to affect the labels.
![screen shot 2017-04-10 at 16 59 34](https://cloud.githubusercontent.com/assets/13964550/24880029/28f1b312-1e0f-11e7-932a-3c0246c7db6d.png)
![screen shot 2017-04-10 at 16 59 24](https://cloud.githubusercontent.com/assets/13964550/24880028/28f0aecc-1e0f-11e7-974f-d1045aafb71c.png)
Any ideas?
Thanks in advance 😊

**Comment by wuxudong** [comment]
you want move label a little right?

**Comment by she-dev** [comment]
In both charts the first label is on the edge of the screen and half of it is lost.
I'd like to fix that but now I'm thinking that moving the label may affect the accuracy of the graph.

The biggest issue to me though is that there's an empty space after the last/rightmost label, before the end of the chart.
Could I force one last label after the one that's being shown to prevent this empty space?

**Comment by she-dev** [comment]
I realized it may be related to scale.
I used green circles around where would be the middle of the label and now it seems like the empty space is due to the next label falling offscreen and therefore not showing up.
![marked3](https://cloud.githubusercontent.com/assets/13964550/24960086/076b5ba0-1f6b-11e7-8542-87021524ea74.png)

[This gif](https://bl.ocks.org/mbostock/raw/2983699/) made me realize that. 

Thanks for your help
